packages/manager/modules/pci/src/projects/project/storages/databases/components/input-rule/input-rule.controller.js

```diff
@@ -0,0 +1 @@
+export default class PciInputRuleController {}
```

Comment:
Needed ?

Reply:
Not anymore, good catch !

Reply:
Changed here: https://github.com/ovh/manager/pull/6263/commits/c1d3c5629b9225d2f11784fb88688fc9b4389a2d